frontend/src/reducers/userReducers.js

```diff
@@ -24,6 +24,18 @@ function userRegisterReducer(state = {}, action) {
     }
 }
 
+function userReloadReducer(state = {}, action) {
+    switch (action.type) {
+        case USER_RELOAD_REQUEST:
+            return { loading: true };
+        case USER_RELOAD_SUCCESS:
+            return { loading: false, userInfo: action.payload };
```

Comment:
I'm unsure if it makes sense that there's 3 userInfo's in store. Should there be just one?

Reply:
Working on this. Thank you so much for this remark. I've been familiar with the concept of redux but never fully understood it. I am much better at it now